addons/account/views/report_invoice.xml

```diff
@@ -225,7 +222,8 @@
             <t t-call="web.html_container">
                 <t t-foreach="docs" t-as="o">
                     <t t-set="lang" t-value="o.invoice_user_id.sudo().lang if o.move_type in ('in_invoice', 'in_refund') else o.partner_id.lang"/>
-                    <t t-call="account.report_invoice_document_with_payments" t-lang="lang"/>
+                    <t t-set="print_with_payments" t-value="True"/>
```

Comment:
We could also integrate the get_custom_report function here and by default it returns the account.report_invoice_document

Reply:
@jco-odoo  hi! ok, I like the idea. I made the push with the requested changes and leave it in a separate commit just for the review. Please let us know what do you think.

Reply:
@zaoral It should be named_get_invoice_report and indeed, the param is not necessary as we can set it through the t-set

Reply:
@jco-odoo it is ok if instead named_get_invoice_report the new name be something like _get_name_invoice_report?